t/cobrand/tfl.t

```diff
@@ -310,6 +314,8 @@ subtest "test report creation anonymously by button" => sub {
 };
 
 subtest "test users with tfl email not asked to update their FMS password" => sub {
+    my $tfl_mock = Test::MockModule->new('FixMyStreet::Cobrand::TfL');
```

Comment:
Do we need this subtest at all any more? If they can't log in this method, this can never happen, so no need to mock it to test it.
(Presumably it should test someone logging in via OIDC doesn't get asked to update their password instead, if that doesn't already get done?)